code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -460,11 +460,11 @@ Congratulations! You are now trained for xenobiology research!"}
 		return
 	var/mob/living/carbon/C = L
 	if(!C.handcuffed)
-		if(C.get_num_arms() >= 2)
+		if(C.get_num_arms() >= 2 || C.get_arm_ignore())
```

Comment:
Alien larvas now only have a chest and head, this is added so they can still be handcuffed despite not having arms.